ams.md

```diff
@@ -0,0 +1,50 @@
+The Anbox Management Service (AMS) sits at the heart of Anbox Cloud and handles all aspects of the application and container life cycle. It is responsible for managing containers, applications, addons, updates and more, ensuring high density, performance and fast container startup times.
+
+AMS is usually managed through the command line interface (the Anbox Management Client - AMC), which can run either on the same machine as AMS or on a remote machine.
+
+Since AMS exposes an HTTP interface, any tool can use the [AMS REST API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801) to interact with AMS. Both the AMC (when running remotely) and the [Anbox Application Registry (AAR)](https://discourse.ubuntu.com/t/application-registry/17761) use this mechanism. You can also develop your own client.
```

Comment:
We maybe should link https://github.com/anbox-cloud/ams-sdk to give users some guidance on how to implement this.